styles/styles.css

```diff
@@ -10,23 +11,24 @@
     --principal-text-color: white;
     --primary-text-color: #121212;
 }
+
 #myBtn {
```

Comment:
You can add negative margin to keep the search bar closer to the top while scrolling.